osu.Server.Queues.ScoreStatisticsProcessor/ScoreStatisticsQueueProcessor.cs

```diff
@@ -160,6 +161,15 @@ protected override void ProcessResult(ScoreItem item)
 
             try
             {
+                SentrySdk.ConfigureScope(scope =>
+                {
+                    scope.SetTag("score_id", item.Score.id.ToString());
+                    scope.User = new SentryUser
+                    {
+                        Id = item.Score.user_id.ToString(),
+                    };
+                });
```

Comment:
I guess this theoretically works as long as the processor is single-threaded but I'm kinda skeeved out by this as the possibility of mislabeling events exists. Could possibly use [something like this](https://github.com/getsentry/sentry-docs/issues/5460#issuecomment-1234516412) instead, dunno.

Reply:
I'm not sure there's a guarantee of single-threadedness.

Just naively assumed sentry would scope per thread but maybe that's not the case. WIll refactor slightly.

Reply:
Note sure `PushScope` is enough because it seems we're running in the default `GlobalMode` which doesn't seems to work well with it, but we'll see.

Reply:
I've tested and the `PushScope()` stuff seems to cause the tags to not be pushed, so force-pushing out.